pkg/utils/watchJob.go

```diff
@@ -83,3 +88,27 @@ func JobLogs(experimentName string, engineName string, client *kubernetes.Client
 	}
 	return 0, nil
 }
+
+//UpdateResultTable will update the result of pipelines in a table by calling a python script result_update.py
+func UpdateResultTable(experimentName string, engineName string, clientSet *chaosClient.LitmuschaosV1alpha1Client) (int, error) {
+
+	//Updating the result table
+	dt := time.Now()
+	app, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
+	if err != nil {
+		return 0, errors.Wrap(err, "Fail to get the chaosresult while updating the result in a table.")
+	}
+	testVerdict := string(app.Status.ExperimentStatus.Verdict)
+	fmt.Println("The job_id for the job will be", os.Getenv("CI_JOB_ID"))
+	fmt.Println("The testVerdict for the experiment will be", testVerdict)
+	cmd := exec.Command("python3", "../utils/result_update.py", "--job_id", os.Getenv("CI_JOB_ID"), "--stage", "Experiment", "--test_desc", experimentName, "--test_result", testVerdict, "--time_stamp", (dt.Format(time.ANSIC)), "--token", os.Getenv("GITHUB_TOKEN"), "--test_name", experimentName)
```

Comment:
Have you written this Python script for update the result, If yes then better to write in same language then you can more control over it, if not then you can change it later

Reply:
This script was written earlier I just modified it.

Reply:
okay!! Can it be converted to Go?

Reply:
I can try that